lib/tests/streamlit/elements/markdown_test.py

```diff
@@ -39,6 +42,43 @@ def test_st_markdown(self):
         self.assertEqual(el.markdown.body, "some markdown")
         self.assertEqual(el.markdown.help, "help text")
 
+    def test_works_with_element_replay(self):
+        """Test that it works with element replay if used as non-widget element."""
```

Comment:
nit: `"""Test that element replay works for non-widget element"""`

Reply:
Updated